spellcheck.json

```diff
@@ -7,6 +7,10 @@
   "musci": "music",
   "musif": "music",
   "musik": "music",
-  "muzeum": "museum"
-  "jewlry": "jewelry"
+  "muzeum": "museum",
+  "jewlry": "jewelry",
+  "clik": "click",
+  "clikc": "click",
+  "cick": "kick",
```

Comment:
```suggestion
```
That could also be `click`.

Reply:
Could be. Another reason why there should be array support…

Reply:
@ScolderCreations That'd require an update to the codebase.